mirte_bringup/launch/minimal.launch

```diff
@@ -11,7 +11,9 @@
          mcu -->
     <node name="mirte_telemetrix_mirte" output="screen"
           pkg="mirte_telemetrix"
-          type="ROS_telemetrix_aio_api.py"/>
+          type="ROS_telemetrix_aio_api.py"
+          respawn="true"
+          respawn_delay="5"/>
```

Comment:
5 secs seems long to me. Including the time for the node to start the whole process can take up to 10s. Did you check if this could be set to 1s or so?

Reply:
This was just a number, I'll change it to 1s after a test

Reply:
Removed delay as it is not required